src/coprocessor/dag/expr/ctx.rs

```diff
@@ -62,6 +62,8 @@ pub struct EvalConfig {
     pub in_select_stmt: bool,
     pub pad_char_to_full_length: bool,
     pub divided_by_zero_as_warning: bool,
+    // TODO: max warning count is not really a EvalConfig. Instead it is a ExecutionConfig, because
```

Comment:
So would we move it and warnings into ExecutionConfig?

Reply:
We don't have ExecutionConfig. This TODO suggests that we can rename EvalConfig to a more precise name ExecutionConfig later if necessary.